src/app/file-manager/reducers/transferables.reducer.ts

```diff
@@ -8,111 +8,114 @@ import { Type } from '@app/file-manager/models/type';
 export type Action = TransferablesActions.All;
 
 export class TransferableState {
-  items: Item[];
+    items: Item[];
```

Comment:
Looks like a re-indent issue; I believe we're using 2 space indent for these files, not 4 space.

Reply:
indeed, we are using 2 space indent, this is now fixed.